docs/data/material/getting-started/templates/landing-page/components/Hero.js

```diff
@@ -40,24 +40,31 @@ export default function Hero() {
               flexDirection: { xs: 'column', md: 'row' },
               alignSelf: 'center',
               textAlign: 'center',
+              fontSize: 'clamp(3rem, 10vw, 4rem)',
```

Comment:
```suggestion
              fontSize: 'clamp(3.5rem, 10vw, 4rem)',
```

I'd bump this up a tiny bit here.